streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilderTest.java

```diff
@@ -1066,6 +1067,41 @@ public void shouldOverrideGlobalStreamsConfigWhenGivenNamedTopologyProps() {
         assertThat(topologyBuilder.topologyConfigs().parseStoreType(), equalTo(Materialized.StoreType.IN_MEMORY));
     }
 
+    @SuppressWarnings("deprecation")
+    @Test
+    public void exceptionHandlerShouldAcceptNewConfig() {
+        final Properties topologyOverrides = new Properties();
+        topologyOverrides.put(StreamsConfig.DEFAULT_DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG, LogAndFailExceptionHandler.class);
+        topologyOverrides.put(StreamsConfig.DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG, LogAndContinueExceptionHandler.class);
+
+        final StreamsConfig config = new StreamsConfig(StreamsTestUtils.getStreamsConfig());
+        final InternalTopologyBuilder topologyBuilder = new InternalTopologyBuilder(
+            new TopologyConfig(
+                "my-topology",
+                config,
+                topologyOverrides)
+        );
+
+        assertThat(topologyBuilder.topologyConfigs().getTaskConfig().deserializationExceptionHandler.getClass(), equalTo(LogAndContinueExceptionHandler.class));
+    }
+
+    @Test
+    public void exceptionHandlerShouldAcceptNewConfigNoOtherDeprecatedConfigPresent() {
```

Comment:
Instead of adding a new test, should we just update an existing test which uses the old config now, and let is use the new config instead (maybe `shouldOverrideGlobalStreamsConfigWhenGivenNamedTopologyProps`) ?